package.json

```diff
@@ -66,30 +66,13 @@
     "vuepress-theme-vue": "^1.1.0",
     "vuex": "^3.1.3"
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "ts",
-      "json"
-    ],
-    "moduleNameMapper": {
-      "^@/(.*)$": "<rootDir>/src/$1",
-      "^test/(.*)$": "<rootDir>/test/$1"
-    },
-    "transform": {
-      "^.+\\.tsx?$": "ts-jest"
-    },
-    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
-    "collectCoverageFrom": [
-      "src/**/*.ts",
-      "!src/index.cjs.ts",
-      "!**/node_modules/**"
-    ],
-    "coverageReporters": [
-      "json",
-      "lcov",
-      "text-summary",
-      "clover"
+  "gitHooks": {
```

Comment:
Can we move up `gitHooks` and `lint-staged` right after `scripts` block? (more of my personal preferences though seems like Vue is doing it).

Reply:
Done :lock: